templates/taxi/manufacturer_list.html

```diff
@@ -1,33 +1,14 @@
 {% extends "base.html" %}
+{% load crispy_forms_filters %}
 
 {% block content %}
-    <h1>Manufacturer List
-    </h1>
+  <h1>{{ object|yesno:"Update, Create" }} Create a Manufacturer</h1>
 
-    {% if manufacturer_list %}
-      <table class="table">
-        <tr>
-          <th>ID</th>
-          <th>Name</th>
-          <th>Country</th>
-        </tr>
+  <form action="" method="post" novalidate>
+    {% csrf_token %}
+    {{ form|crispy }}
+    {{ form.as_p }}
```

Comment:
The form is being rendered twice: once with `{{ form|crispy }}` and again with `{{ form.as_p }}`. Choose one method to render the form to avoid duplication. If you are using crispy forms, `{{ form|crispy }}` is sufficient.